src/components/admin/registrations/RegistrationsTeamsTable.tsx

```
@@ -0,0 +1,594 @@
+          <span className="text-xs text-muted-foreground">
+            {displayMembers.length} / {MAX_SLOTS}
+          </span>
+        </div>
+        <div className="flex gap-2 px-4 py-4">
+          {displayMembers.map(member => (
+            <MemberCard key={member.registration_id} member={member} onRemove={onDeleteMember} />
+          ))}
+          {!filter &&
+            Array.from({ length: emptySlots }).map((_, i) => (
+              <EmptySlot
+                key={`empty-slot-${i}-${emptySlots}`}
```

Biome (2.4.14)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)